src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,53 @@
+package duke.task;
+
+/**
+ * Tasks are used in the entire program.
+ */
+public abstract class Task {
+    /** Task description. **/
+    private String description;
+
+    /** Status of task. **/
+    private boolean isDone;
+
+    public Task(String description) {
```

Comment:
The initial design was to avoid the default false parameter from propagating throughout the program as it seemed unnecessary. Thanks for the suggestion!